### PR TITLE
fix render hook footer in simple layout (login,registration..) pages

### PR DESCRIPTION
**Pull request by atmonshi**
- [x] Changes have been thoroughly tested to not break existing functionality.
- [x] New functionality has been documented or existing documentation has been updated to reflect changes.
- [x] Visual changes are explained in the PR description using a screenshot/recording of before and after.

when adding a `renderHook` for the footer:
```php
->renderHook(
    'panels::footer',
    fn(): View => view('filament.hooks.footer'),
)
```

the hook will show on the side 

fila2

Following the same markup structure from the layout `resources/views/components/layout/index.blade.php`
I moved the footer to the `fi-simple-main-ctn` container.

fila1

the issue appears in a fresh filament v3 installation and has also been tested on other pages too (login, registration, and reset password)

### CODE REVIEW

**Comment by zepfietje** [comment]
The render hook should stay in place.
Probably just need to add `flex-col` to the wrapping `div`?

Mark as ready for review when done and tested. 🙂

**Comment by atmonshi** [comment]
@zepfietje it got complecated :)

Adding `flex-col`, the form won't be centered anymore, 

Screenshot 2023-08-03 at 5 05 19 PM


So I added `flex items-center flex-grow` to the `fi-simple-main-ctn` div to make it center again

Screenshot 2023-08-03 at 5 05 52 PM

on small screen
Screenshot 2023-08-03 at 5 18 06 PM

What do you think?

**Comment by atmonshi** [comment]
This is a bit more refactored, but I think it looks better

Screenshot 2023-08-03 at 5 31 56 PM

after login

Screenshot 2023-08-03 at 5 32 44 PM

**Comment by zepfietje** [comment]
Hmm I think the previous screenshots looked better; when the footer is pushed to the bottom.

**Comment by atmonshi** [comment]
prefect, tested and ready :)

File: packages/panels/resources/views/components/layout/simple.blade.php [hunk 1]
```diff
@@ -18,7 +18,7 @@ class="absolute end-0 top-0 flex h-16 items-center gap-x-4 pe-4 md:pe-6 lg:pe-8"
             </div>
         @endif
 
-        <div class="fi-simple-main-ctn w-full">
+        <div class="fi-simple-main-ctn w-full flex items-center flex-grow">
```
Comment (zepfietje) [inline packages/panels/resources/views/components/layout/simple.blade.php]: Are `flex` and `items-center` needed here?

**Comment by atmonshi** [comment]
> So I added `flex items-center flex-grow` to the `fi-simple-main-ctn` div to make it center again

yup otherways won't center the form on the page

File: packages/panels/resources/views/components/layout/simple.blade.php [hunk 2]
```diff
@@ -18,7 +18,7 @@ class="absolute end-0 top-0 flex h-16 items-center gap-x-4 pe-4 md:pe-6 lg:pe-8"
...
             <main
                 class="fi-simple-main mx-auto my-16 w-full bg-white px-6 py-12 shadow-sm ring-1 ring-gray-950/5 dark:bg-gray-900 dark:ring-white/10 sm:max-w-lg sm:rounded-xl sm:px-12"
```
Comment (zepfietje) [inline packages/panels/resources/views/components/layout/simple.blade.php]: Could you try removing `mx-auto` and test if everything still works as expected? I think that class is no longer needed now we use flex for the parent, @atmonshi.

Comment (atmonshi) [reply packages/panels/resources/views/components/layout/simple.blade.php]: then I have to add `justify-center` to the parent :) to center it 

Screenshot 2023-08-04 at 4 27 03 PM

Comment (zepfietje) [reply packages/panels/resources/views/components/layout/simple.blade.php]: Oh I see, well both work.

**Comment by zepfietje** [comment]
Thanks

**Pull request merged**